src/components/common/styles.scss

```diff
@@ -0,0 +1,171 @@
+$theme-orange: linear-gradient(to right, #ff512f 0%, #f09819 51%, #ff512f 100%);
+$theme-orange-dark: linear-gradient(
+	to right,
+	#e04627 0%,
+	#cc7a15 51%,
+	#e04627 100%
+);
+$theme-carrot: #f28d1c;
```

Comment:
these vars move to a single file like, var.scss. And move it to styles folder at root

Reply:
<img width="167" alt="image" src="https://github.com/thomasdinh2k/FintTestReact-LoginScreen/assets/130777277/1cabb4e2-c830-4559-a118-7339a7fd78b7">

var has been encapsulated under _"src/styles/_variables.scss"_